swatch-contracts/send_amqp_message.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
```

Comment:
This file should be moved to the `config/artemis/` folder or `bin`.

Reply:
I have moved to `config/artemis`